users/views/_admin_menu_edit.php

```diff
@@ -14,7 +14,7 @@
 foreach ($index as $i) {
   $indexes[] = $i->z_index;
 }
-$parentOpts = $db->query("SELECT * FROM us_menu_items WHERE menu = ? AND `type` = ? ORDER BY label", [$menuId, "dropdown"])->results();
+$parentOpts = $db->query("SELECT * FROM us_menu_items WHERE menu = $1 AND \"type\" = $2 ORDER BY label", [$menuId, "dropdown"])->results();
```

Comment:
What is the problem with positional `?` placeholders in a PDO request with PostgreSQL?

Reply:
So both work but here I was having a SQL error when using the ? placeholder. The AI suggested using the $ placeholder so the error was gone. The query was working also. Can it be a different problem? Should I keep it or revert it?

Reply:
I reverted other placeholders too in this file